daemon/mgr/image.go

```diff
@@ -340,6 +343,17 @@ func (mgr *ImageManager) CheckReference(ctx context.Context, idOrRef string) (ac
 	return
 }
 
+// ListReferences returns all references
+func (mgr *ImageManager) ListReferences(ctx context.Context, imageID digest.Digest) ([]reference.Named, error) {
+	refs := mgr.localStore.GetPrimaryReferences(imageID)
+	if len(refs) == 0 {
+		err := errtypes.ErrNotfound
+		logrus.Errorf("one Image ID must have the primary references, but got nothing")
```

Comment:
@xiaoxubeii The imageID might be invalid or non-existing. I don't think we should return the non-found error here. how do you think?